cmd/wordrow/main.go

```diff
@@ -48,9 +48,11 @@ func main() {
 	if shouldRun {
 		setLogLevel(args)
 
-		err := run(args)
-		if err != nil {
-			logger.Error(err)
+		errs := run(args)
+		if len(errs) > 0 {
+			for _, err := range errs {
```

Comment:
This is getting nested pretty deep, consider refactoring

Reply:
Luckily, the `if len(errs) > 0` check is unnecessary, the for-loop won't do anything in that case anyway :smile: